sympy/ntheory/digits.py

```diff
@@ -21,8 +21,34 @@ def digits(n, b=10):
     [2, 1, 1, 0, 1, 1]
     >>> digits(65536, 256)
     [256, 1, 0, 0]
-    >>> digits(-3958, 27)
```

Comment:
please see my previous edits and raise OverflowError if the number of number of requested digits is less than the number needed (i.e. b**(digits - 1) <= n).

It is ok to use the name `digits` instead of `bits`:
```python
>>> def digits(n,digits=None):
...  print(n,digits)
... 
>>> digits(3,4)
(3, 4)
```

Reply:
Okay sir. Working on it.